EmbeddedSocialClient/sdk/src/main/java/com/microsoft/embeddedsocial/server/model/notification/GetNotificationFeedResponse.java

```diff
@@ -17,21 +17,22 @@
 public class GetNotificationFeedResponse extends FeedUserResponse implements ListResponse<ActivityView> {
 
 	private List<ActivityView> activities;
+	// Setting this handle is required to properly update the "last-read" notification
 	private String deliveredActivityHandle;
 
 	public GetNotificationFeedResponse(List<ActivityView> activities) {
 		this.activities = activities;
-
-		// FIXME fix setting this handle
-		this.deliveredActivityHandle = "";
+		this.deliveredActivityHandle = activities.get(0).getHandle();
```

Comment:
Although unlikely, it is possible for the server to return an empty list of activities. We should add a check that the list of activities is not empty, to avoid a null dereference.

Reply:
Fixed in next commit!